src/js/internal/util/inspect.js

```diff
@@ -133,6 +133,16 @@ const {
   Uint8Array,
 } = primordials;
 
+const kCustomPromisifyArgsSymbol = Symbol('customPromisifyArgs');
```

Comment:
This util/inspect.js file is 92 KB of JS

Please move this function into a separate file, so that it can be lazily loaded without loading all of util/inspect.js

<img width="58" alt="image" src="https://github.com/user-attachments/assets/80726dd3-a6a5-4645-869a-83f4e2d049da">

Reply:
edit: it looks like we are already loading this file in util.js 

so it's probably fine